core/options.c

```diff
@@ -2443,6 +2442,8 @@ options_init()
     /* .lspdata pages start out writable so no unprotect needed here */
     write_lock(&options_lock);
     ASSERT(sizeof(dynamo_options) == sizeof(options_t));
+    /* Set to default options. */
+    dynamo_options = default_options;
```

Comment:
Seems cleaner to use set_dynamo_options_defaults(), and then remove the call to adjust_defaults_for_page_size() below.

Reply:
Done.